triangle.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func main() {
+	a := 0.0
+	b := 0.0
+	c := 0.0
```

Comment:
в таком случае принято объявлять переменные через ```var```
+ в задании сказано, что числа целые, поэтому int

```suggestion
	var a int
	var b int
	var b int
```